cosmoz-default-tree.html

```diff
@@ -146,6 +146,10 @@
 
 	/**
 	 * Returns the nodes on a given path.
+	 * A valid path 1.2.3 should return the items [1, 2, 3]
+	 * - path 1.2.3.3 should return [1, 2, 3, undefined]
+	 * - path 0.1.2.3 should return [1, 2, 3]
+	 * - path 0.1.5.3 should return [1, undefined, undefined]
```

Comment:
Imagine this path:
```
1 - 9 - 2 - 3
  - 8 - 2 - 3
  - 7 - 2 - 3
```
We are looking for `0.1.5.2`, which one is the correct path? Not sure, but I think when it's lost it should be lost? hm..